### ISSUE TITLE
long @INC list causes exec fail

### ISSUE THREAD

**Issue by plicease** [plicease/Test-Script#5]
Noticed this coming in from cpantesters for `Dist::Zilla::Plugin::Test::Version` :

```
t/00-report-prereqs.t .. ok

#   Failed test 'check test compiles'
#   at t/file.t line 47.
# 72057594037927935 - 
# exception: Argument list too long at /usr/home/cpan/pit/thr/conf/perl-5.16.2/.cpanplus/5.16.2/build/hS8qgVCJx9/Test-Script-1.10/blib/lib/Test/Script.pm line 96.
# signal: 127

#   Failed test 'check test runs'
#   at t/file.t line 48.
# 72057594037927935 - 
# exception: Argument list too long at /usr/home/cpan/pit/thr/conf/perl-5.16.2/.cpanplus/5.16.2/build/hS8qgVCJx9/Test-Script-1.10/blib/lib/Test/Script.pm line 126.
# signal: 127
# Looks like you failed 2 tests of 6
```

http://www.cpantesters.org/cpan/report/b950a0f2-0c50-11e6-829f-59a5aef69d38

**Comment by plicease** [comment]
after releasing `1.11_02` I got this cpantesters result:

http://www.cpantesters.org/cpan/report/117d9dcc-0ca4-11e6-a144-6a0e9f4cbc8b

so far the only fail.  May need to do some error checking on the writing of that temporary .pm file.

**Comment by plicease** [comment]
Assuming this change makes it to a prod version `Dist::Zilla::Plugin::Test::Version` should update its prereqs because it seems to be getting hit repeatedly on cpantesters by this.  I am guessing it has a ridiculously long dependency list.

**Comment by plicease** [comment]
fixed in 1.12
